boxes/identity-server-analytics/confs/repository/conf/datasources/master-datasources.xml

```diff
@@ -0,0 +1,110 @@
+<datasources-configuration xmlns:svns="http://org.wso2.securevault/configuration">
```

Comment:
BTW, shall we decouple Configs from been burnt into the boxes and make them available with default configs ?
We can easily mount any deployment specific configs later when we are doing that specific deployment.

Reply:
+1